### PR TITLE
[ISSUE #1824] Improve test coverage of org.apache.shenyu.admin.service.*

### PR DESCRIPTION
**Pull request by JooKS-me**
Ref #1824 .

### CODE REVIEW

**Comment by 0x12FD16B** [comment]
plz resolve conflict first.

**Comment by JooKS-me** [comment]
> plz resolve conflict first.

ok... a tricky question

File: shenyu-admin/src/test/java/org/apache/shenyu/admin/service/UpstreamCheckServiceTest.java [hunk 1]
```diff
@@ -66,15 +90,53 @@
...
+            Object[] args = invocationOnMock.getArguments();
+            if (MOCK_SELECTOR_NAME.equals(args[0])) {
+                return selectorDOWithUrlError;
+            } else if (MOCK_SELECTOR_NAME_OTHER.equals(args[0])) {
+                return selectorDOWithUrlReachable;
+            }
+            return null;
+        });
+        when(selectorConditionMapper.selectByQuery(any(SelectorConditionQuery.class))).thenReturn(Collections.emptyList());
+        doNothing().when(eventPublisher).publishEvent(any(DataChangedEvent.class));
```
Comment (dengliming) [inline shenyu-admin/src/test/java/org/apache/shenyu/admin/service/UpstreamCheckServiceTest.java:137]: These codes already removed by [https://github.com/apache/incubator-shenyu/pull/1843](https://github.com/apache/incubator-shenyu/pull/1843). Please check again. Thanks.

Comment (JooKS-me) [reply shenyu-admin/src/test/java/org/apache/shenyu/admin/service/UpstreamCheckServiceTest.java]: @dengliming Thanks for reminding. Yep, I check it, there are some unnecessary stubbings, and I delete it. But, some should be left, or the coverage will drop.

Comment (dengliming) [reply shenyu-admin/src/test/java/org/apache/shenyu/admin/service/UpstreamCheckServiceTest.java]: I mean we should remove all unnecessary stubbings.

Comment (JooKS-me) [reply shenyu-admin/src/test/java/org/apache/shenyu/admin/service/UpstreamCheckServiceTest.java]: @dengliming ok, I have refactored it, please have a review again. There should be no unnecessary stubbings in UpstreamCheckServiceTest.

**Review by dengliming — APPROVED** [review]

**Pull request merged**